mkdocs.yml

```diff
@@ -83,10 +84,10 @@ nav:
   - usage/dataclasses.md
   - usage/validation_decorator.md
   - usage/postponed_annotations.md
-  - Errors:
-    - Usage Errors: usage/errors.md
-    - Validation Errors: usage/validation_errors.md
   - usage/conversion_table.md
+- Error Messages:
+  - Validation Errors: usage/validation_errors.md
```

Comment:
Don't we need to have a separate directory for error messages and move these two files there?

Reply:
Navigation is not dependent on the location of the files. 

Does moving them break any existing links from errors to the docs?

Reply:
> Does moving them break any existing links from errors to the docs?

I don't think but I am not sure